.github/actions/publish-dbt-artifacts/publish.py

```
@@ -22,12 +22,20 @@
+    print(f"{manifest} not found - skipping publish")
+    sys.exit(0)
 
+# Record the engine that produced the artifacts (unpinned installs = latest).
 fusion_version = None
-if os.path.exists(".fusion-version"):
-    with open(".fusion-version") as f:
-        fusion_version = f.read().strip()
+try:
+    import subprocess
+
+    out = subprocess.run(["dbt", "--version"], capture_output=True, text=True, timeout=30).stdout
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)